.github/workflows/batch-arm-merge-queue.yml

```
@@ -0,0 +1,34 @@
+
+      - name: Get PRs
+        id: list-prs
+        run: PRS=$(gh pr list --repo "$GITHUB_REPOSITORY" --state open --json number,draft --limit 100 | jq -r '.[] | select(.draft == false) | .number'); echo "prs=$PRS" >> $GITHUB_OUTPUT
+
+      - name: Arm PRs
+        if: inputs.dry_run == false
+        run: for pr in ${{ steps.list-prs.outputs.prs }}; do gh pr merge --auto --merge "https://github.com/$GITHUB_REPOSITORY/pull/$pr" --repo "$GITHUB_REPOSITORY" || true; done; echo "Done!"
+
+      - name: Dry run
+        if: inputs.dry_run == true
+        run: echo "DRY RUN: Would arm ${{ steps.list-prs.outputs.count }} PRs"
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)